examples/analog_in/cont_thrmcpl_samples_int_clk.py

```diff
@@ -0,0 +1,29 @@
+﻿"""Example of continuous temperature acquisition.
+
+This example demonstrates how to make continuous, hardware-timed
+temperature measurement using a thermocouple.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, ThermocoupleType, TemperatureUnits
+
+with nidaqmx.Task() as task:
+    task.ai_channels.add_ai_thrmcpl_chan(
+        "Dev1/ai0", units=TemperatureUnits.DEG_C, thermocouple_type=ThermocoupleType.K
```

Comment:
let's also specify cjc_source and cjc_val even through we'll hardcode the defaults. it at least gives folks some breadcrumbs for how they might configure the cjc.

```suggestion
        "Dev1/ai0", units=TemperatureUnits.DEG_C, thermocouple_type=ThermocoupleType.K, cjc_source=CJCSource.CONSTANT_USER_VALUE, cjc_val=25.0
```